src/client/src/apps/MainRoute/widgets/spotTile/selectors.ts

```diff
@@ -1,19 +1,59 @@
 import { createSelector } from 'reselect'
 import { GlobalState } from 'StoreTypes'
-import { SpotTileContainerOwnProps } from './SpotTileContainer'
+import { getDefaultNotionalValue } from './components/Tile/TileBusinessLogic';
+import { SpotTileData, SpotTileDataWithNotional } from './model';
+import { CurrencyPair } from 'rt-types';
+import { PriceMovementTypes } from './model/priceMovementTypes';
 
-const getCurrencyPair = (state: GlobalState, props: SpotTileContainerOwnProps) =>
-  state.currencyPairs[props.id]
+const DEFAULT_TILE_DATA: SpotTileData = {
```

Comment:
these defaults were lifted from view - this is to make sure that we would not duplicate such code across multiple views